File: 02 - Cpp/04 - Crescente.cpp
#include <iostream>

using namespace std;

int main()
{
	int primeiro, segundo;

	cout << "Digite dois valores: " << '\n';
	cin >> primeiro;
	cin >> segundo;

	while (primeiro != segundo)
	{
		if (primeiro > segundo)
		{
			cout << "DECRESCENTE" << '\n';
		}
		else
		{
			cout << "CRESCENTE" << '\n';
		}
		
		cout << "Digite outros dois valores: " << '\n';
		cin >> primeiro;
		cin >> segundo;
	}

	return 0;
}

File: 03 - Java/04 - Crescente.java
import java.util.Scanner;

public class Main 
{
	public static void main(String[] args) 
	{		
		Scanner sc = new Scanner(System.in);
		
		int primeiro, segundo;
		
		System.out.println("Digite dois numeros: ");
		primeiro = sc.nextInt();
		segundo = sc.nextInt();
		
		while (primeiro != segundo)
		{
			if (primeiro < segundo)
			{
				System.out.println("CRESCENTE");
			}
			else
			{
				System.out.println("DECRESCENTE");
			}
			
			System.out.println("Digite outros dois numeros: ");
			primeiro = sc.nextInt();
			segundo = sc.nextInt();
		}
		
		sc.close();
	}
}
